persistence/persistence-jpa/pom.xml

```diff
@@ -38,25 +38,94 @@
         <type>pom</type>
         <scope>import</scope>
       </dependency>
+      <dependency>
+        <groupId>org.springframework.boot</groupId>
+        <artifactId>spring-boot</artifactId>
+        <version>2.1.6.RELEASE</version>
```

Comment:
Please use the dependency manager in pom instead of specify the version directly here.
It will help us manage the third party dependencies.